contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-epilogue.y.c

```diff
@@ -692,6 +692,49 @@ is_json_query(List *name)
 	}
 }
 
+static Node *
```

Comment:
Please add header comments to all functions, even static functions, that summarize what they do.

Reply:
Done